tests/model/model.cpp

```diff
@@ -552,3 +554,47 @@ TEST(Model, setAndCheckIdsAllEntities)
     const std::string a = printer->printModel(m);
     EXPECT_EQ(a, e);
 }
+
+TEST(Model, missingUnitsFromImportOfCnTerms)
+{
+    std::string a =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+        "<model xmlns=\"http://www.cellml.org/cellml/2.0#\" name=\"myModel\">\n"
+        "   <units name=\"myUnitsThatReallyExist\">\n"
+        "       <unit exponent=\"-1\" units=\"second\"/>\n"
+        "   </units>\n"
+        "   <component name=\"myComponent\">\n"
+        "       <variable name=\"a\" units=\"second\"/>\n"
+        "       <math xmlns=\"http://www.w3.org/1998/Math/MathML\" xmlns:cellml=\"http://www.cellml.org/cellml/2.0#\">\n"
+        "           <apply><eq/>\n"
+        "               <ci>a</ci>\n"
+        "               <cn cellml:units=\"myUnitsThatReallyExist\">1</cn>\n"
+        "           </apply>\n"
+        "       </math>\n"
+        "   </component>\n"
+        "</model>";
+    auto parser = libcellml::Parser::create();
+    auto modelFromString = parser->parseModel(a);
+    auto validator = libcellml::Validator::create();
+    validator->validateModel(modelFromString);
+    EXPECT_EQ(size_t(0), validator->errorCount());
+
+    // Import the membrane component from a file
```

Comment:
Not "critical" for tests, but would still add a full stop at the end of the sentence.

Reply:
Done